project/tests.py

```diff
@@ -0,0 +1,149 @@
+            2003, 7, 14, tzinfo=datetime.timezone.utc))
+        self.assertEqual(self.project_update.ong, self.ong2)
+
+    ### TESTS UPDATE PROJECT ###
+
+    def test_project_update_title_incorrect_max_length(self):
+        with self.assertRaises(Exception):
+            self.project_update.title = "T" * 101
+            self.project_update.full_clean()
+            self.project_update.save()
+
+    def test_project_update_title_incorrect_null(self):
+        with self.assertRaises(Exception):
+            self.project_update.title = None
+            self.project_update.full_clean()
+            self.project_update.save()
+
+    def test_project_update_title_incorrect_blank(self):
+        with self.assertRaises(Exception):
+            self.project_update.title = ""
+            self.project_update.full_clean()
+            self.project_update.save()
+
+    def test_project_update_country_incorrect_max_length(self):
+        with self.assertRaises(Exception):
+            self.project_update.country = "C" * 101
+            self.project_update.full_clean()
+            self.project_update.save()
+
+    def test_project_update_start_date_incorrect(self):
+        with self.assertRaises(Exception):
+            self.project_update.start_date = "incorrectDate"
+            self.project_update.full_clean()
+            self.project_update.save()
+
+    def test_project_update_end_date_incorrect(self):
+        with self.assertRaises(Exception):
+            self.project_update.end_date = "incorrectDate"
+            self.project_update.full_clean()
+            self.project_update.save()
+
+    def test_project_update_end_date_before_start_date(self):
+        with self.assertRaises(Exception):
+            self.project_update.start_date = datetime.datetime(
+                2003, 7, 14, tzinfo=datetime.timezone.utc)
+            self.project_update.end_date = datetime.datetime(
+                1990, 7, 14, tzinfo=datetime.timezone.utc)
+            self.project_update.full_clean()
+            self.project_update.save()
+
+    def test_project_update_number_of_beneficiaries_incorrect(self):
+        with self.assertRaises(Exception):
+            self.project_update.number_of_beneficiaries = "incorrectNumber"
+            self.project_update.full_clean()
+            self.project_update.save()
+
+    def test_project_update_number_of_beneficiaries_incorrect_min(self):
+        with self.assertRaises(Exception):
+            self.project_update.number_of_beneficiaries = -1
+            self.project_update.full_clean()
+            self.project_update.save()
+
+    def test_project_update_amount_incorrect(self):
+        with self.assertRaises(Exception):
+            self.project_update.amount = "incorrectNumber"
+            self.project_update.full_clean()
+            self.project_update.save()
+
+    def test_project_update_amount_incorrect_min(self):
+        with self.assertRaises(Exception):
+            self.project_update.amount = -1
```

Comment:
I think here you should update the amount to 0 instead of -1 because the minimum amount defined by the validator is 1. What do you think about this?